infra/ansible/roles/build_srcs/tasks/main.yaml

```diff
@@ -76,12 +76,12 @@
     pushd /tmp/staging-wheels
     cp {{ item.dir }}/*.whl .
     # First rename to add '.dev'
-    rename -v "s/^{{ item.prefix }}-(.*?)(\+.*?)-cp/{{ item.prefix }}-\1.dev-cp/" *.whl
+    rename -v 's/^{{ item.prefix }}-(.*?)(\+.*?)(-cp.*$)/{{ item.prefix }}-\1.dev\3/' *.whl
     mv /tmp/staging-wheels/* /dist/
     popd
 
     # Second rename to append the date in YYYYMMDD format
-    rename -v "s/^{{ item.prefix }}-(.*?)-cp/{{ item.prefix }}-\1.dev$(date -u +%Y%m%d)-cp/" *.whl
+    rename -v "s/^{{ item.prefix }}-(.*?)(\+.*?)(-cp.*$)/{{ item.prefix }}-\1.dev$(date -u +%Y%m%d)\3/" *.whl
```

Comment:
I don't fully understand why `-cp` is replaced with `\3/`. what's the `(\+.*?)` trying to match?

Can we just add a comment in the yaml file to indicate the origional whl name and what it suppose to update it to?

Reply:
This one probably tells better about the grouping: https://regex101.com/r/5Uldrj/1. Let me add details in the file.